ingredient/ingredient-postgresql/src/SingleServerArm.json

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
That was included in the quickstart template. https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-postgresql-server-database-using-arm-template?tabs=azure-portal. I ended up deleting this file at Mo's suggestion since it's dead code. We could probably get a template without the metadata fields using the Azure Portal downloadable template thing, if we really want.